src/main/java/org/myrobotlab/service/InMoov2Torso.java

```diff
@@ -93,7 +93,7 @@ public void disable() {
       lowStom.disable();
   }
   
-  public void onMoveTorso(HashMap<String, Double> map) {
+  public void onMove(Map<String, Double> map) {

```

Comment:
backwards compatibility should be maintained here..  add this method.. mark the old one as deprecated.. make the old one call this one instead.

Reply:
:+1: